src/pages/vault/downloads/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement } from 'react'
+import { GetStaticProps } from 'next'
+import vaultData from 'data/waypoint.json'
```

Comment:
OMG I led you wrong. So sorry! Copy-paste error 🤣

```suggestion
import vaultData from 'data/vault.json'
```